File: soal1.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
      Lion l = new Lion();
      l.makeSound();
      
      System.out.println();
      
      Elephant e = new Elephant();
      e.makeSound();
  }
}

public class Animal{
  private  String name;
  private int age;
  
  public void makeSound(){
    System.out.print("test");
  }
}

public class Lion extends Animal{
  public void makeSound(){
    System.out.print("Roar");
  }
}

public class Elephant extends Animal{
  public void makeSound(){
    System.out.print("Trumpet");
  }
}

File: soal1.php
<?php

class Animal {
    public $name;
    public $age;

    function makeSound() {
        echo "test";
    }
}

class Lion extends Animal{
    function makeSound(){
        echo "Roar";
    }
}

class Elephant extends Animal{
    function makeSound(){
        echo "Trumpet";
    }
}

$l = new Lion();
$l->makeSound();
echo "\n";
$e = new Elephant();
$e->makeSound()
?>